platforms/org.eclipse.mita.platform.xdk110/1.0.0/xdk110.platform

```diff
@@ -815,8 +814,6 @@ bus named-singleton GPIO {
 	signal digitalOut(pin : GPIOPin, initialValue : bool = false) : bool
 }
 
-exception EndOfFileException
-
```

Comment:
See comment below; I recommend keeping this.

Reply:
Added exception back